containers/Makefile.am

```diff
@@ -0,0 +1,39 @@
+
+KUBERNETES_DIR = $(srcdir)/containers/kubernetes
+
+CLEANFILES += $(KUBERNETES_DIR)/rpms
+
+kubernetes-container:
+	rm -rf $(KUBERNETES_DIR)/rpms && mkdir -p $(KUBERNETES_DIR)/rpms
+	cd $(KUBERNETES_DIR)/rpms && ../../../tools/make-rpms
+	sudo docker build -t cockpit/kubernetes $(KUBERNETES_DIR)
```

Comment:
I'm fine with adding that in a separate pull request as an option. That option didn't exist prior to this time. These are new make targets.